### PR TITLE
OCPBUGS-19517: remove unsupported commands from microshift docs

### PR DESCRIPTION
**Pull request by dhellmann**
We missed some of the commands that are not supported on MicroShift in
the last review. This change removes commands related to deployment
configs and image streams. It also removes the debug command, which
needs a release payload to determine the toolbox image.

### CODE REVIEW

File: tools/gendocs/gendocs/microshift.go [hunk 1]
```diff
@@ -66,11 +67,24 @@ var microshiftCommands = sets.NewString(
 	"oc registry info",
 	"oc registry login",
 	"oc replace",
+	"oc rollback",
+	"oc rollback",
```
Comment (ardaguclu) [inline tools/gendocs/gendocs/microshift.go:71]: nit: `oc rollback` appears 2 times.

**Review by ardaguclu — COMMENTED** [review]
/lgtm

File: tools/gendocs/gendocs/microshift.go (hunk 1 above)
Comment (dhellmann) [reply tools/gendocs/gendocs/microshift.go]: Since CI failed I went ahead and fixed this one.

**Pull request merged**